app/assets/javascripts/darkswarm/services/enterprise_registration_service.js.coffee

```diff
@@ -11,7 +11,11 @@ Darkswarm.factory "EnterpriseRegistrationService", ($http, RegistrationService,
       for key, value of enterpriseAttributes
         @enterprise[key] = value
 
-    create: =>
+    # Creates the enterprise and redirects to the about step on success.
+    #
+    # @param callback [Function] executed at the end of the operation both in
+    #   case of success or failure.
```

Comment:
I couldn't find any example of function documentation in darkwswarm so I used the first doc format I found on google for coffee script 👉  https://github.com/coffeedoc/codo